desc/backend.py

```diff
@@ -184,6 +194,62 @@ def tree_unstack(tree):
         leaves, treedef = jtu.tree_flatten(tree)
         return [treedef.unflatten(leaf) for leaf in zip(*leaves)]
 
+    _eigvals_cpu = jax.jit(jnp.linalg.eigvals, device=jax.devices("cpu")[0])
+
+    @jax.custom_jvp
+    def eigvals(A):
```

Comment:
Not sure, but would it be worth to add a unit test for this and the _eigvals_jvp functions? I know they just wrap JAX but they would be easily tested on a small 2x2 matrix or something